### PR TITLE
Feature styles

### PR DESCRIPTION
**Pull request by matheus-fls**
Add calculator styling.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,67 @@
...
   color: white;
+  font-size: 20px;
+  font-weight: bold;
+  justify-content: right;
+  align-items: center;
+  padding: 12px;
+  width: 100%;
+}
+
+.button-panel {
```
Comment (Oghenebrume50) [inline src/App.css:37]: This should use flex box
screenshot

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED

Hi, @matheus-fls good start to this one, please make changes as outlined below and we are good. 
**please make changes and request for a re-review** :slightly_smiling_face: 

**RAPHAEL NORIODE** :sunglasses:

File: src/App.css (hunk 1 above)
Comment (matheus-fls) [reply src/App.css:37]: Done!

**Review by shloch — APPROVED** [review]
## Hello @matheus-fls 

You can merge this, this is a thumbs up from me

See you on next milestone
Happy coding

**Pull request merged**